packages/@react-spectrum/list/src/List.tsx

```diff
@@ -0,0 +1,149 @@
+        padding: 0, // TODO: get from DNA
+        collator
+      })
+    , [collator, scale]);
+
+  layout.collection = state.collection;
+  layout.disabledKeys = state.disabledKeys;
+  return layout;
+}
+
+function List<T extends object>(props, ref: DOMRef<HTMLDivElement>) {
+  let {
+    transitionDuration = 0
+  } = props;
+  let domRef = useDOMRef(ref);
+  let {collection} = useListState(props);
+
+  let gridCollection = new GridCollection({
+    columnCount: 1,
+    items: [...collection].map(item => ({
+      type: 'row',
+      childNodes: [{
+        ...item,
+        index: 0,
+        type: 'cell'
+      }]
+    }))
+  });
+
+
+  let state = useGridState({
+    ...props,
+    collection: gridCollection
+  });
+  let layout = useListLayout(state); // TODO should layout return nulls?
+
+  let collator = useCollator({usage: 'search', sensitivity: 'base'});
+  let {direction} = useLocale();
+  let keyboardDelegate = new GridKeyboardDelegate({
+    collection: state.collection,
+    disabledKeys: state.disabledKeys,
+    ref: domRef,
+    direction,
+    collator,
+    focusMode: 'cell'
+  });
+  let {gridProps} = useGrid({
+    keyboardDelegate,
+    ref: domRef
+  }, state);
+
+  let {styleProps} = useStyleProps(props);
+
+  // This overrides collection view's renderWrapper to support hierarchy of items in sections.
+  // The header is extracted from the children so it can receive ARIA labeling properties.
+  type View = ReusableView<Node<T>, unknown>;
+  let renderWrapper = (parent: View, reusableView: View) => (
+    <VirtualizerItem
+      key={reusableView.key}
+      reusableView={reusableView}
+      parent={parent} />
+  );
+
+  return (
+    <ListContext.Provider value={{state, keyboardDelegate}}>
+      <Virtualizer
+        {...gridProps}
+        {...styleProps}
+        ref={domRef}
+        focusedKey={state.selectionManager.focusedKey}
+        sizeToFit="height"
+        scrollDirection="vertical"
+        className={
+          classNames(
+            listStyles,
+            'react-spectrum-List',
+            styleProps.className
+          )
+        }
+        layout={layout}
+        collection={collection}
+        renderWrapper={renderWrapper}
+        transitionDuration={transitionDuration}>
+        {(type, item) => {
+          if (type === 'item')  {
+            return (
+              <ListItem item={item} />
+            );
+          } else if (type === 'loader') {
+            return (<div>wait circle</div>);
+          } else if (type === 'placeholder') {
+            let emptyState = props.renderEmptyState ? props.renderEmptyState() : null;
+            if (emptyState == null) {
+              return null;
+            }
+
+            return (
+              <div
+                // aria-selected isn't needed here since this option is not selectable.
+                // eslint-disable-next-line jsx-a11y/role-has-required-aria-props
+                role="option">
```

Comment:
This should not be an option because it's not inside a listbox. Should be a `<div role="row"><div role="gridcell">{emptyState}</div></div>` because it's in a grid.